inventory/osinfo/osinfo_linux.go

```diff
@@ -104,24 +103,22 @@ func GetDistributionInfo() (*DistributionInfo, error) {
 	case util.Exists(rhRelease):
 		releaseFile = rhRelease
 		parseReleaseFunc = parseEnterpriseRelease
-	default:
-		err = errors.New("unable to obtain release info, no known /etc/*-release exists")
 	}
 
 	b, err := ioutil.ReadFile(releaseFile)
 	if err != nil {
-		di = &DistributionInfo{ShortName: Linux}
+		oi = &OSInfo{ShortName: Linux}
 	} else {
-		di = parseReleaseFunc(string(b))
+		oi = parseReleaseFunc(string(b))
 	}
 
-	out, err := getUname()
```

Comment:
does getUname still need to exist, then?

Reply:
Nope, removed